client/test/cypress/support/fragments/demoproject.js

```diff
@@ -0,0 +1,12 @@
+export const reset = () => {
+  cy.get('body').then(body => {
+    if (body.find('[data-nrt="deleteDemo"]').length >  0) {
+      cy.GetByDataNrt('deleteDemo').click()
+      // TODO : Replace the basic Modal component of iviewui by a custom one to have
```

Comment:
Why not replace TODOs by new issues ?

Reply:
Done.

See : https://github.com/Decathlon/ara/issues/224
(Issue number also added in the TODO).